File: DossierUT5/React/TypeScript/my-app/src/Persona.ts

 class Persona {

    id: number;
    nombre: string;
    apellido: string;
    altura: number;
    edad: number;
    peso: number;

    constructor(){
        this.id = 0;
        this.nombre= "";
        this.apellido = "";
        this.altura =0;
        this.edad =0;
        this.peso = 0;
    }

    calcular() {
        return this.peso / (this.altura*this.altura);       
    }



}
export default Persona ;

File: Dossier UT1/es.iespuertodelacruz.activdadEspecial/src/es/iespuertodelacruz/activdadespecial/model/Persona.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package es.iespuertodelacruz.activdadespecial.model;

/**
 *
 * @author Adrián Rodríguez Fuentes
 */
public class Persona {

    private String nombre;
    private String apellido;
    private String edad;

    public Persona() {
    }

    public Persona(String nombre, String apellido, String edad) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.edad = edad;
    }

    public String getNombre() {
        return nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public String getEdad() {
        return edad;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public void setEdad(String edad) {
        this.edad = edad;
    }

    @Override
    public String toString() {
        return nombre +" "+ apellido +" "+  edad ;
    }
    
    

    
}
